tests/pytests/test_stats.py

```diff
@@ -181,14 +186,181 @@ def testIssue1497(env):
     for i in range(count):
         env.expect('DEL', 'doc%d' % i)
 
-    for _ in range(50):
-        forceInvokeGC(env, 'idx')
+    forceInvokeGC(env, 'idx')
 
-    res = env.cmd('ft.info', 'idx')
-    d = {res[i]: res[i + 1] for i in range(0, len(res), 2)}
-    env.assertEqual(d['inverted_sz_mb'], '0')
-    env.assertEqual(d['num_records'], 0)
-    check_empty(env, 'idx')
+    # The memory occupied by a empty TEXT and TAG inverted index is 
+    # 54 bytes * doc_count, becase FGC_applyInvertedIndex() is calling 
+    # InvertedIndex_AddBlock() for each delete doc.
+    # The memory occupied by a empty NUMERIC and GEO inverted index is 102 bytes.
+    expected_inverted_sz_mb = (54 * 2 * count) / (1024 * 1024) \
+                    + (102 * 2) / (1024 * 1024)
+    check_empty(env, 'idx', expected_inverted_sz_mb)
+
+@skip(cluster=True)
+def testMemoryAfterDrop_numeric(env):
+
+    if env.cmd('FT.CONFIG', 'GET', 'GC_POLICY')[0][1] != 'fork':
+        env.skip()
+
+    idx_count = 100
+    doc_count = 120
+    divide_by = 1000000   # ensure limits of geo are not exceeded
+    pl = env.getConnection().pipeline()
+
+    env.execute_command('FLUSHALL')
+    env.execute_command('ft.config', 'set', 'FORK_GC_CLEAN_THRESHOLD', 0)
+
+    for i in range(idx_count):
+        env.expect('FT.CREATE', 'idx%d' % i, 'PREFIX', 1, '%ddoc' % i, 'SCHEMA', 'n', 'NUMERIC').ok()
+
+    for i in range(idx_count):
+        geo = '1.23456,' + str(float(i) / divide_by)
+        for j in range(doc_count):
+            pl.execute_command('HSET', '%ddoc%d' % (i, j), 'n', i)
+        pl.execute()
+        d = index_info(env, 'idx%d' % i)
+        env.assertEqual(d['num_docs'], doc_count)
+
+    for i in range(idx_count):
+        for j in range(doc_count):
+            pl.execute_command('DEL', '%ddoc%d' % (i, j))
+        pl.execute()
+        d = index_info(env, 'idx%d' % i)
+        env.assertEqual(d['num_docs'], 0)
+        forceInvokeGC(env, 'idx%d' % i)
+
+    # The memory occupied by a empty NUMERIC inverted index is 102 bytes,
+    # which is the sum of the following (See NewInvertedIndex()):
+    # sizeof_InvertedIndex(index->flags)   48
+    # sizeof(IndexBlock)                   48
+    # INDEX_BLOCK_INITIAL_CAP               6
+    expected_inverted_sz_mb = 102 / (1024 * 1024)
+    for i in range(idx_count):
+        check_empty(env, 'idx%d' % i, expected_inverted_sz_mb)
+
+@skip(cluster=True)
+def testMemoryAfterDrop_geo(env):
+
+    if env.cmd('FT.CONFIG', 'GET', 'GC_POLICY')[0][1] != 'fork':
+        env.skip()
```

Comment:
since it is repeated, consider adding a skip rule

Reply:
OK. A new rule added `@skip(gc_no_fork=true)`